x/compliance/keeper/msg_server_certify_model.go

```diff
@@ -76,6 +76,10 @@ func (k msgServer) CertifyModel(goCtx context.Context, msg *types.MsgCertifyMode
 	} else {
 		// There is no compliance record yet. So certification will be tracked on ledger.
 
+		if !types.IsValidPFCCertificationRoute(msg.ParentChild) {
```

Comment:
Please move this check to `validate` in message_certify_model (the same way as for `IsValidCertificationType`)

Reply:
Done! ✅